### PR TITLE
feat: New wallet address format – misc

### PR DESCRIPTION
**Pull request by igordanilcenko**
- Replaced short 3-digit emojis in texts with the new format
- Removed unused code
- Hid edit Yat field in the edit contact dialog by the feature flag

### CODE REVIEW

File: app/src/main/java/com/tari/android/wallet/ui/fragment/chat/chatList/adapter/ChatItemViewHolder.kt [hunk 1]
```diff
@@ -28,7 +28,7 @@ class ChatItemViewHolder(view: ItemChatItemBinding) : CommonViewHolder<ChatItemV
             } else {
                 firstEmojiTextView.visible()
                 avatar.gone()
-                firstEmojiTextView.text = item.firstEmoji
+                firstEmojiTextView.text = "a"
```
Comment (TruszczynskiA) [inline app/src/main/java/com/tari/android/wallet/ui/fragment/chat/chatList/adapter/ChatItemViewHolder.kt:31]: a?

File: app/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -270,7 +270,7 @@
     <string name="add_recipient_scan_qr_error">Invalid QR Code.\nYour request cannot be processed at this time.</string>
     <string name="add_recipient_failed_init_camera_message">Failed to initialize Camera</string>
     <string name="add_recipient_camera_permission_denied_message">Camera permission denied</string>
-    <string name="qr_code_scanner_labels_actions_transaction_send">This code contains a funds request.\nWould you like to send funds to %s?</string>
+    <string name="qr_code_scanner_labels_actions_transaction_send">This code contains a funds request.\nWould you like to send funds to \n%s?</string>
```
Comment (TruszczynskiA) [inline app/src/main/res/values/strings.xml:273]: QR phrase is missing here and in localisable keys bellow.

File: app/src/main/java/com/tari/android/wallet/ui/fragment/chat/chatList/adapter/ChatItemViewHolder.kt (hunk 1 above)
Comment (igordanilcenko) [reply app/src/main/java/com/tari/android/wallet/ui/fragment/chat/chatList/adapter/ChatItemViewHolder.kt:31]: @TruszczynskiA it's just a placeholder for the chat item. Just wanted to remove round avatar

File: app/src/main/res/values/strings.xml (hunk 2 above)
Comment (igordanilcenko) [reply app/src/main/res/values/strings.xml:273]: @TruszczynskiA What do you mean exactly?

File: app/src/main/java/com/tari/android/wallet/ui/fragment/chat/chatList/adapter/ChatItemViewHolder.kt (hunk 1 above)
Comment (TruszczynskiA) [reply app/src/main/java/com/tari/android/wallet/ui/fragment/chat/chatList/adapter/ChatItemViewHolder.kt:31]: Why you need a placeholder here?

**Review by TruszczynskiA — APPROVED** [review]

**Pull request merged**